src/ActionsImporter/Commands/Bamboo/Common.cs

```diff
@@ -40,13 +40,13 @@ public static class Common
         IsRequired = false,
```

Comment:
Is the `--include-from` cli option hooked up in Bamboo? IIRC, this is a no-op for all providers except CircleCI.

Reply:
Good catch! It is not an option we use for Bamboo. 

Removed in https://github.com/github/gh-actions-importer/pull/133/commits/27cb6f3be13b23859aafd170afb33ce03e2066f6